taxi/forms.py

```diff
@@ -0,0 +1,48 @@
+import re
+
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+
+from taxi.models import Driver, Car
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    class Meta:
+        model = Driver
+        fields = ["license_number"]
+
+    def clean_license_number(self):
+        license_number = self.cleaned_data["license_number"]
+        if not re.match(r"^[A-Z]{3}[0-9]{5}$", license_number):
+            raise ValidationError("License number should starts with"
+                                  " 3 uppercase letters and 5 digits")
+        return license_number
+
+
+class DriverCreationForm(UserCreationForm):
+    class Meta(UserCreationForm.Meta):
+        model = Driver
+        fields = UserCreationForm.Meta.fields + ("license_number",
+                                                 "first_name",
+                                                 "last_name", )
+
+    def clean_license_number(self):
+        license_number = self.cleaned_data["license_number"]
+        if not re.match(r"^[A-Z]{3}[0-9]{5}$", license_number):
+            raise ValidationError("License number should starts with"
+                                  " 3 uppercase letters and 5 digits")
```

Comment:
There is a minor grammatical error in the validation error message. It should be 'License number should start with 3 uppercase letters and 5 digits' instead of 'starts'.